### PR TITLE
boards: fanstel_ev832x : add board support for Fanstel EV-BT832X

### PR DESCRIPTION
**Pull request by cvinayak**
Add support for Fanstel EV-BT832X board.

Signed-off-by: Vinayak Kariappa Chettimada <vinayak.chettimada@sixoctets.com>

### CODE REVIEW

**Comment by cvinayak** [comment]
@clidleo FYI

File: boards/arm/fanstel_ev832x/doc/fanstel_ev832x.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.. _fanstel_ev832x:
+
+Fanstel EV-BT832X
+#################
+
+Overview
+********
+
+BluNor BT832X is a powerful, highly flexible, Bluetooth Low Energy (BLE) using
```
Comment (dbkinder) [inline boards/arm/fanstel_ev832x/doc/fanstel_ev832x.rst]: This doesn't read as an overview of the EV-BT832X development board. Maybe this:
```
The Fanstel EV-BT832X board is an evaluation board for the BluNor BT832X Bluetooth Low Energy module.
This board uses a Nordic nRF52832 SoC, with an ARM Cortex M4F MCU, 512KB flash,
64KB RAM, embedded 2.4GHz multi-protocol transceiver, power amplifier, and an
integrated PCB trace antenna, or an u.FL connector for external antenna.
```

File: boards/arm/fanstel_ev832x/doc/fanstel_ev832x.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Nordic nRF52832 SoC. With an ARM Cortex (TM) M4F MCU, available 512KB flash,
+64KB RAM, embedded 2.4GHz multi-protocol transceiver, power amplifier, and an
+integrated PCB trace antenna, or an u.FL connector for external antenna. It
+allows faster time to market with reduced development cost.
+
+For applications needing limited number of IO pins, prototyping and production
```
Comment (dbkinder) [inline boards/arm/fanstel_ev832x/doc/fanstel_ev832x.rst]: I'd delete these next two paragraphs and let folks reference the material you've linked to for more information.

But you do need information about how to build a Zephyr app for this board and how to run it (check out another board document for an example).

**Review by dbkinder — CHANGES_REQUESTED** [review]
Board documentation needs additional work.

File: boards/arm/fanstel_ev832x/doc/fanstel_ev832x.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+The Fanstel EV-BT832X board is an evaluation board for the BluNor BT832X
+Bluetooth Low Energy module. This board uses Nordic Semiconductor nRF52832 SoC,
+with an ARM Cortex (TM) M4F MCU, available 512KB flash, 64KB RAM, embedded
+2.4GHz multi-protocol transceiver, power amplifier, and an integrated PCB trace
+antenna, or an u.FL connector for external antenna.
+
+See `fanstel ev-bt832x website`_ for more information about the development
+board and `nRF52832 website`_ for the official reference on the IC itself.
+
```
Comment (dbkinder) [inline boards/arm/fanstel_ev832x/doc/fanstel_ev832x.rst]: Good so are... but still missing board usage information typical of our board docs (see, for example, https://docs.zephyrproject.org/latest/boards/arm/nucleo_l053r8/doc/nucleol053r8.html)

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by galak** [comment]
Any plans to update?

**Comment by cvinayak** [comment]
> 
> 
> Any plans to update?

@galak This has been in my backlog for sometime, this is a board that I used for manually validating controllers feature (PA/LNA support). This is not on top of my priority task, but when I get to regression testing the feature and use the board, I will send updates to this PR. 

I think I would visit this PR targeting 2.1 release

**Comment by ioannisg** [comment]
> I think I would visit this PR targeting 2.1 release

Shall we move this to 2.2 release now?

**Comment by cvinayak** [comment]
> Shall we move this to 2.2 release now?

Yes

**Comment by cvinayak** [comment]
@dbkinder regret the enormous delay on follow up! I have updated the PR now, please take a look.

**Comment by galak** [comment]
Closing this as this board port hasn't been updated in over a year.  Feel free to update and re-open.

**Pull request closed without merging**